jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -117,8 +117,13 @@ public Integer call() {
         logger.log(LogEvent.Level.WARN, "--jvm-flags is ignored when --entrypoint is specified");
       }
 
-      CacheDirectories cacheDirectories =
-          CacheDirectories.from(commonCliOptions, jarFile.toAbsolutePath().getParent());
+      Path jarFileParentDir = jarFile.toAbsolutePath().getParent();
```

Comment:
Too bad it doesn't recognize this can never fail because of `toAbsolutePath()`. Maybe just throw `RuntimeException("BUG: getParent() null)` instead of logging.

Reply:
Oh, actually, what we've been doing is
```suggestion
      Path jarFileParentDir = Verify.verifyNotNull(jarFile.toAbsolutePath().getParent());
```
This should please NullAway.

Reply:
How about we use `verifyNotNull` instead?

Reply:
Will do!